app/js/collections/todos.js

```diff
@@ -0,0 +1,10 @@
+define([
+  'backbone',
+  'models/todo'
+], function (Backbone, ToDo) {
+  var TodoCollection = Backbone.Collection.extend({
+    model: ToDo
```

Comment:
incostintent naming, you either change `ToDo` to `Todo` and `TodoCollection` to `ToDoCollection`